smartlearner/interfaces/loss.py

```diff
@@ -5,20 +6,29 @@ class Loss(object):
     def __init__(self, model, dataset):
         self.model = model
         self.dataset = dataset
-        self.target = dataset.symb_targets
         self.consider_constant = []  # Part of the computational graph to be considered as a constant.
+        self.graph_updates = OrderedDict()
+        self._gradients = None
+
+    @property
+    def target(self):
+        return self.dataset.symb_targets
 
     def get_graph_output(self):
         output, updates = self.model.get_model_output(self.dataset.symb_inputs)
```

Comment:
Is that a problem? 

On August 15, 2015 9:21:49 PM EDT, "Marc-Alexandre Côté" <notifications@github.com> wrote:
>>          self.consider_constant = []  # Part of the computational
>graph to be considered as a constant.
>> +        self.graph_updates = OrderedDict()
>> +        self._gradients = None
>> +
>> +    @property
>> +    def target(self):
>> +        return self.dataset.symb_targets
>>  
>>      def get_graph_output(self):
>>          output, updates =
>self.model.get_model_output(self.dataset.symb_inputs)
>
>However, this implies that every subclass must now use
>`self.dataset.symb_targets` directly.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/SMART-Lab/smartlearner/pull/31/files#r37140995

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.